gensim/test/test_translation_matrix.py

```diff
@@ -102,17 +102,21 @@ def test_translation_matrix(self):
         transmat = model.train(self.train_docs[:5])
         self.assertEqual(transmat.shape, (8, 8))
 
-    def test_infer_vector(self):
+    def disabled_test_infer_vector(self):
```

Comment:
I think it's better to use the built-in mechanism to skip tests.

```suggestion
    @unittest.skip("skip reason")
    def test_infer_vector(self):
```

Reply:
Addressed in latest commit.